paddle/fluid/framework/ir/mkldnn/fc_elementwise_add_mkldnn_fuse_pass.cc

```diff
@@ -93,9 +94,18 @@ GraphWithStats FCResidualConnectionMKLDNNFusePass::FuseFC(
     GET_IR_NODE_FROM_SUBGRAPH(
         elementwise_out, elementwise_out, elementwise_pattern);
 
-    if (FindFuseOption(*fc_op, *elementwise_op) != FUSE_MKLDNN) return;
-    if (!IsReachable(g, residual_data, fc_output)) return;
-    if (HasFusedActivation(fc_op)) return;
+    if (FindFuseOption(*fc_op, *elementwise_op) != FUSE_MKLDNN) {
+      VLOG(4) << "Skipping fusion because not both ops have use_mkldnn";
+      return;
+    }
+    if (!IsReachable(g, residual_data, fc_output)) {
+      VLOG(4) << "Skipping fusion because residual input is not reachable";
+      return;
+    }
+    if (HasFusedActivation(fc_op)) {
+      VLOG(4) << "Skipping fusion because fc_op has activation fused";
+      return;
+    }
```

Comment:
It would be great if you could include op names in logs. it will make it easier to debug.

Reply:
done